### PR TITLE
Implement SauceLabs web integration

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: vividus-plugin-web-app/src/main/resources/properties/profile/profile.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+selenium.grid.idle-timeout=360
+selenium.grid.selenium-version=3.12.0
```
Comment (ikalinin1) [inline vividus-plugin-web-app/src/main/resources/properties/profile/profile.properties]: at last, why 3.12.0?

File: vividus-plugin-web-app/src/main/resources/properties/profile/saucelabs/web/profile.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+selenium.grid.capabilities.sauce\:options.idleTimeout=${selenium.grid.idle-timeout}
+selenium.grid.capabilities.sauce\:options.seleniumVersion=${selenium.grid.selenium-version}
+selenium.grid.capabilities.sauce\:options.screenResolution=${selenium.grid.screen-resolution}
+selenium.grid.capabilities.sauce\:options.commandTimeout=180
```
Comment (ikalinin1) [inline vividus-plugin-web-app/src/main/resources/properties/profile/saucelabs/web/profile.properties]: Why the command timeout changed?

File: vividus-plugin-web-app/src/main/resources/properties/profile/profile.properties (hunk 1 above)
Comment (valfirst) [reply vividus-plugin-web-app/src/main/resources/properties/profile/profile.properties]: it's legacy, the removal of this capability should be done in a separate PR (as you did it) and ideally after this one to test all browsers

File: vividus-plugin-web-app/src/main/resources/properties/profile/saucelabs/web/profile.properties (hunk 2 above)
Comment (uarlouski) [reply vividus-plugin-web-app/src/main/resources/properties/profile/saucelabs/web/profile.properties]: sauce labs specific options should be places under sauce\:options as per documentation, this prop doesn't influence other grids

Comment (ikalinin1) [reply vividus-plugin-web-app/src/main/resources/properties/profile/saucelabs/web/profile.properties]: it was 300 now it's 180

File: vividus-plugin-web-app/src/main/resources/properties/profile/web/tablet/profile.properties [hunk 3]
```diff
@@ -1,7 +1,6 @@
 bdd.all-meta-filters=groovy: layout && layout.toString().tokenize(' ').contains('tablet') && !skip && (${bdd.meta-filters})
 selenium.grid.capabilities.deviceOrientation=portrait
 
-selenium.grid.capabilities.commandTimeout=300
```
[the remark this answers is not in the document]
Comment (uarlouski) [reply vividus-plugin-web-app/src/main/resources/properties/profile/web/tablet/profile.properties]: - this is not selenium capability, but rather saucepans specific, so selenium standalone simply doesn't recognize that
- as to tablet and mobiles profiles that use appium another property should be used called newCommandTimeout

File: vividus-plugin-web-app/src/main/resources/properties/profile/saucelabs/web/profile.properties (hunk 2 above)
Comment (uarlouski) [reply vividus-plugin-web-app/src/main/resources/properties/profile/saucelabs/web/profile.properties]: It was 300 for tables and phones (in appium this props should be newCommandTimeout), for web it was 180

**Pull request merged**